src/diffusers/pipelines/pag/pipeline_pag_sana.py

```diff
@@ -185,6 +189,7 @@ def encode_prompt(
         clean_caption: bool = False,
         max_sequence_length: int = 300,
         complex_human_instruction: Optional[List[str]] = None,
+        lora_scale: Optional[float] = None,
```

Comment:
Are we training text encoder? If not, we can remove these changes maybe

Reply:
This was to have no surprise for our users when text encoder training support is merged. It's common to see the `encode_prompt()` method being equipped with handling `lora_scale`.